Makefile.in

```diff
@@ -172,11 +175,29 @@ uninstall:
 	@rm -rf -v --preserve-root=all $(DESTDIR)$(DATADIR)/$(INSTALLDIR)
 
 
-.PHONY: user-install
-user-install:
-	$(info :: --- )
-	$(info :: Installed under user-only location "$(DATADIR)/$(INSTALLDIR)" )
-	$(info :: To run you need to make sure "$(BINDIR)" is in your PATH )
+ZIPAPP := $(OBJDIR)/umu-run
+ZIPAPP_STAGING := $(OBJDIR)/umu-run_staging
+
+$(OBJDIR)/.build-zipapp: | $(OBJDIR)
+	$(info :: Building umu-launcher as zipapp )
+	$(PYTHON_INTERPRETER) -m pip install -t "$(ZIPAPP_STAGING)" -U --no-compile .
+	cp umu/umu_version.json.in umu/umu_version.json.in.tmp
+	sed 's|##UMU_VERSION##|$(shell git describe --always --long --tags)|g' -i umu/umu_version.json.in.tmp
+	mv umu/umu_version.json.in.tmp "$(ZIPAPP_STAGING)"/umu/umu_version.json
```

Comment:
This could be achieved with the `version` target instead for consistency. Why is it separate?